### PR TITLE
add project: pavilion

### PR DESCRIPTION
**Pull request by baasdesign**

### CODE REVIEW

**Comment by jimaek** [comment]
You need an update.json file as well. Check our README for more info

**Comment by baasdesign** [comment]
I see, I am creating one now! :)

**Comment by baasdesign** [comment]
Done

**Comment by baasdesign** [comment]
Is it OK like this?

File: files/pavilion/update.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "pavilion",
+  "repo": "getpavilion/pavilion",
+  "files": {
+    "basePath": "/dist",
```
Comment (jimaek) [inline files/pavilion/update.json]: `dist/`

File: files/pavilion/update.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    "include": ["pavilion.css", "pavilion.min.css", "pavilion.min.css.map"]
```
Comment (jimaek) [inline files/pavilion/update.json]: Remove `pavilion.css` its not needed in production

**Comment by baasdesign** [comment]
Done

**Comment by jimaek** [comment]
You didnt update the dist path

**Comment by baasdesign** [comment]
Sorry, my bad

**Comment by baasdesign** [comment]
thanks

**Pull request merged**